examples/config-search.json

```diff
@@ -12,10 +12,7 @@
     },
     "extensions": {
         "search": {
-          "enable": true,
```

Comment:
This is not correct. Existing configurations should work as is. Add a new example if need be called config-remote-only.json